src/app/lib/config/toggles/liveConfig.js

```diff
@@ -29,6 +29,9 @@ export default {
   navOnArticles: {
     enabled: true,
   },
+  nielsenAnalytics: {
```

Comment:
I'll need to add this new toggle in our toggles UI

Reply:
OK thanks. I wondered if/where an external versions of the toggles were!

Reply:
@TimHewitt I know it's stated in ticket but is adding a toggle for this absolutely necessary? No worries if it. Just don't want to add and maintain another toggle if we probably won't toggle this feature on/off

Reply:
Sorry @jroebu14 missed this...
I think it is required due to potential GDPR issues, but will double check...

Reply:
From John Horth: 
_My understanding was that we should always have a means to readily disable third party integrations. In this instance where user data is being handled, and global legislation constantly evolving that req was amplified._

Reply:
I believe @aeroplanejane has now set this toggle up 👍